content/home/participate.md

```diff
@@ -18,11 +18,19 @@ Also feel free to add yourself and/or your organization to
 This is a public listing for those individual and company contributors
 who declared their interest in the project.
 
+## Slack
+
+We plan to create the official Slack workspace later so that multiple workstreams and channels could be established there.
+For now, we have an `$openfeature` Slack channel on the CNCF Slack Workspace.
+You can join this Slack using [this link](https://slack.cncf.io/).
+
 ## Project meetings
 
 Stay tuned for the announcements!
+**[Sign-up for news](https://bit.ly/openfeature-signup)** to get invite to the time poll and the meetings
```

Comment:
This sentence is a bit confusing. Could you please rephrase it?

Reply:
```suggestion
**[Sign-up for news](https://bit.ly/openfeature-signup)** to get invited the meetings.
```